spec/models/policies/early_years_payments/policy_eligibility_checker_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+describe Policies::EarlyYearsPayments::PolicyEligibilityChecker do
+  subject { described_class.new(answers: answers) }
+
+  describe "#status, #ineligible?, #ineligibility_reason" do
+    context "when ineligible as :returner" do
+      let(:answers) do
+        build(
+          :early_years_payment_answers,
+          returning_within_6_months: true,
+          returner_worked_with_children: true,
+          returner_contract_type: "permanent"
+        )
+      end
+
+      it "is ineligble as :returner" do
+        expect(subject).to be_ineligible
+        expect(subject.status).to eql(:ineligible)
+        expect(subject.ineligibility_reason).to eql(:returner)
+      end
```

Comment:
I think we're missing scenarios here for when `returning_within_6_months` and `returner_worked_with_children` are false.
Currently, if the PolicyEligibilityChecker is refactored to remove any 2 out of 3 checks, the spec still passes.